### DISCUSSION TITLE
Tag for gameplay differences of remakes/clones

### DISCUSSION THREAD

**Discussion by Hiradur** [opengaming/osgameclones#1384] [Ideas]
I think it's a good idea to write a few sentences about every remake/clone on how they differ from the original game. This would make it easier for people to find the right game for them. This could also be put under the `info` tag but maybe that tag should be kept for other information.

Here are a few examples made up from my own research (e.g. forum posts by others):
```
0ad:
Focus on accurate representation of ancient warfare. Most notable difference to Age of Empires is the ability of most war units being usable as regular workers. This has big implications on gameplay.

JSettlers:
Adds additional controls not found in The Settlers III. Stageless zoom and fine control over game speed is available. AI is stronger than in the original.

Megaglest:
Base game comes with 7 factions. Units are less durable so micro-management seems to be a bit less important than in Warcraft 3. Various mods are available however, some trying to bring competitive play closer to Warcraft 3.

Rigs of Rods:
Focus on building a versatile soft-body physics sandbox. "A hardcore simulator for a hardcore audience" as the lead developer put it. More than 3000 vehicles and maps available. Unlike BeamNG a multiplayer mode is available. Hauling, rock crawling, mudding and roleplay seem to be popular in the community.

Return to The Roots:
Adds another faction (Babylonians) to the already existing factions in The Settlers II. Faithful to the original game in the default configuration, gameplay features can be added/altered with an addon manager.

Simutrans:
In contrast to Transport Tycoon Deluxe every good has its own destination. There are a lot more options to build train tracks. Accurate models of train tracks and schedules of whole countries can be created and have already been done by community members.

Widelands:
Even more focus on economy than The Settlers II. More goods and more complex logistic chains. Buildings can be upgraded to higher tiers.
```

**Comment by sansolovyov** [comment]
Like, `description`? :)

**Comment by Hiradur** [comment]
> Like, description? :)

Something like this, although `gameplay-description` may be a bit more accurate but also very long.

**Comment by sansolovyov** [comment]
Yeah, it's too long for sure. :)

**Comment by Hiradur** [comment]
`gameplay` maybe?

**Comment by sansolovyov** [comment]
That one sounds good! Go ahead! :)

**Comment by tukkek** [comment]
I don't think that's a good idea. The list isn't a place to soapbox about games, it is a clean and objective resource and should be kept that way. The links are there so interested players can follow them and read everything about the clones in more detail on the proper pages (and not on the list itself).

**Comment by Hiradur** [comment]
The reason I suggested the `gameplay` tag is because information on gameplay is sometimes hard to find. But I agree that it can be very subjective and I'm now leaning towards not implementing it to keep the list objective as you said.

**Comment by tukkek** [comment]
On that note some games already have descriptions on the current version of the list. For example Caesar 3, and Cube 2: Sauerbraten clones. I'd like to reiterate that this doesn't seem to be the proper role for this project and this kind of information is very likely to become outdated since we don't have people constantly checking it for accuracy as the projects get developed further on. 

If others agree, would they also be in favor of such information being removed? The mere fact for me that only a few of these games have text associated with them, while they're not any more or less special than all the other games that have no text in them is a big sign for me that this is out of place.

**Comment by cxong** [comment]
I think using the info field is fine for now.
